nominal/core/stream.py

```diff
@@ -20,7 +20,7 @@
 @dataclass(frozen=True)
 class BatchItem:
     channel_name: str
-    timestamp: IntegralNanosecondsUTC
+    timestamp: datetime | IntegralNanosecondsUTC
```

Comment:
I don't think we should turn this into a union, should just be integral ns and we should fix anywhere that doesn't use that

Reply:
OK, will fix up!